test/functional/normal/tabpage_spec.lua

```diff
@@ -0,0 +1,38 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local clear = helpers.clear
+local command = helpers.command
+local eq = helpers.eq
+local feed = helpers.feed
+local eval = helpers.eval
+
+describe('ui/tabpage', function()
```

Comment:
- can remove `ui/` prefix here since it's not a UI test
- file should have 2-space indent

Reply:
Fixed.